lib/src/request.dart

```diff
@@ -7,15 +7,14 @@ import 'package:api_utils/src/status_code.dart';
 import 'package:api_utils/src/timeout.dart';
 import 'package:http/http.dart' as http;
 import 'package:meta/meta.dart';
-import 'package:sentry_flutter/sentry_flutter.dart';
 
 @visibleForTesting
 void setHttpClientForTesting(http.Client client) {
   _client = client;
 }
 
-void setSentryHttpClient({SentryHttpClient? client}) {
-  _client = client ?? SentryHttpClient(client: http.Client());
+void setHttpClient(http.Client client) {
+  _client = client;
 }
```

Comment:
These methods do the same thing, no need to have both. As I mentioned in my PR feedback, we could rename the original method and remove the annotation instead of duplicating.